modules/aws/core/route53.tf

```diff
@@ -7,8 +7,10 @@ resource "aws_route53_zone" "this" {
     if var.dns_zone_enabled
   }
 
-  tags = {
-    Name        = var.dns_zone
-    Environment = var.environment
-  }
+  tags = merge(
+    local.global_tags,
+    {
+      Name        = var.dns_zone
+    },
+  )
```

Comment:
Any point in using default tags? https://registry.terraform.io/providers/hashicorp/aws/latest/docs#default_tags

Reply:
Default tags only work in the provider so I would rather not.